doc/user-guide/io.rst

```diff
@@ -566,29 +566,12 @@ This is not CF-compliant but again facilitates roundtripping of xarray datasets.
 Invalid netCDF files
 ~~~~~~~~~~~~~~~~~~~~
 
-The library ``h5netcdf`` allows writing some dtypes (booleans, complex, ...) that aren't
+The library ``h5netcdf`` allows writing some dtypes that aren't
 allowed in netCDF4 (see
-`h5netcdf documentation <https://github.com/shoyer/h5netcdf#invalid-netcdf-files>`_).
+`h5netcdf documentation <https://github.com/h5netcdf/h5netcdf#invalid-netcdf-files>`_).
 This feature is available through :py:meth:`DataArray.to_netcdf` and
 :py:meth:`Dataset.to_netcdf` when used with ``engine="h5netcdf"``
-and currently raises a warning unless ``invalid_netcdf=True`` is set:
-
-.. ipython:: python
-    :okwarning:
-
-    # Writing complex valued data
```

Comment:
Maybe we should keep the example and highlight the new `auto_complex=True` of netcdf4-python?

Reply:
I think it would be very useful to have an example.

Is it still true that a warning is raised? Is that warning xarray or h5netcdf?

Reply:
There will be no warning raised if capable versions are detected.

Reply:
There is a notice in the howdoi.rst how to handle this, but we might have to add the capable version numbers there.

Reply:
Extensive example will be put in the tutorial.